RELEASE-NOTES.txt

```diff
@@ -1,3 +1,7 @@
+21.2
+-----
+* [*] Removed WeeklyRoundup notifications as part of Jetpack focus [#19590]
```

Comment:
The release note will need to be changed, since the code logic changed as well.

There's also a question what and if anything needs to be mentioned p1668600572541539-slack-C047UD1KHNE